package.json

```diff
@@ -1,67 +1,18 @@
 {
-  "name": "ethereum-waffle",
-  "description": "Sweeter, faster and simpler than truffle.",
-  "version": "2.2.0",
-  "author": "Marek Kirejczyk <account@ethworks.io> (http://ethworks.io)",
-  "repository": "git@github.com:EthWorks/Waffle.git",
-  "private": false,
-  "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.ts",
-  "types": "dist/esm/index.d.ts",
-  "bin": {
-    "waffle": "./bin/waffle"
-  },
-  "scripts": {
-    "prepublishOnly": "yarn build",
-    "test:buildonly": "ts-node script/buildTestContracts",
-    "test:nobuild": "export NODE_ENV=test && mocha",
-    "test": "yarn test:buildonly && yarn test:nobuild",
-    "lint": "eslint '{lib,test}/**/*.ts'",
-    "lint:fix": "eslint --fix '{lib,test}/**/*.ts'",
-    "build": "rimraf ./dist && yarn build:esm && yarn build:cjs",
-    "build:esm": "tsc -p tsconfig.build.json --outDir dist/esm --module ES6",
-    "build:cjs": "tsc -p tsconfig.build.json --outDir dist/cjs --declaration false",
-    "clean": "rimraf ./dist ./test/compiler/build ./test/example/build ./test/matchers/build"
-  },
+  "name": "waffle-monorepo",
+  "private": true,
   "engines": {
     "node": ">=10.0"
   },
-  "dependencies": {
-    "@resolver-engine/imports": "^0.3.3",
-    "@resolver-engine/imports-fs": "^0.3.3",
-    "@types/ganache-core": "^2.7.0",
-    "ethers": "^4.0.0",
-    "ganache-core": "^2.8.0",
-    "mkdirp": "^0.5.1",
-    "solc": "^0.5.10"
-  },
-  "resolutions": {
-    "web3": "1.2.1"
+  "workspaces": [
+    "waffle-*"
+  ],
+  "scripts": {
+    "lint": "wsrun -c lint",
+    "build": "wsrun -te -c build",
+    "test": "wsrun -c test"
```

Comment:
Would be nice to have "publish" here as well.

Reply:
Publishing is tricky. There are egde cases (what if one package has no changes? What if a package has changes, but the version is not bumped). I agree that it would be nice, but I see it as a future improvement and not a part of this pr.